qualtran/bloqs/qsp/polynomial_approximations.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright 2024 Google LLC
```

Comment:
I think we should move this file to `qualtran/linalg/` directory instead of having it here in `qualtran/bloqs/qsp/`. The reason is that I imagine we'll be adding more such linalg utility methods that can be used to approximate functions as polynomials. One example we already have is `get_inverse_square_root_poly_coeffs` which approximates the inverse sqrt-root function as a polynomial which is the used in Bloqs for function approximations using QROM. 

Tl;Dr - Please move the file to `qualtran/linalg/` and maybe give a more specific name like `approx_exp_by_jacobi_anger.py`

Reply:
moved